run-ci.sh

```diff
@@ -0,0 +1,20 @@
+FIRST_COMMIT_HASH=$( git log master..$( git branch --show-current ) --pretty=oneline | tail -1 | awk '{ print $1 }' )
+LAST_COMMIT_HASH=$( git rev-parse HEAD )
+ARTICLES=$( git diff --diff-filter=d --name-only ${FIRST_COMMIT_HASH}^ ${LAST_COMMIT_HASH} "wiki/**/*.md" "news/*.md" )
+
+echo -e "\n--- Binary file size check ---\n"
+bash .github/scripts/ci/inspect_binary_file_sizes.sh ${FIRST_COMMIT_HASH} ${LAST_COMMIT_HASH}
+
+echo -e "\n--- Run remark ---\n"
+bash .github/scripts/ci/run_remark.sh ${FIRST_COMMIT_HASH} ${LAST_COMMIT_HASH}
+
+echo -e "\n--- Run yamllint ---\n"
+python .github/scripts/ci/run_yamllint.py --config .yamllint.yaml
```

Comment:
I think we need a small piece of code which will bootstrap the environment if it doesn't exist, like

```sh
pip3 install venv
python3 -m venv ./.venv
source ./.venv/bin/activate
pip install -r requirements.txt
```

(and also do not care about 2.x since it's expired)

Reply:
are you implying this should be in-lined in the script?

Reply:
I just noticed that we have at least one undeclared dependency (`yamllint`, see frenzibyte@'s comment) and decided to add a note here

as for inlining.. likely not, because we probably need the same for node

Reply:
I've added a section in the contributing file. Very open to suggestions here